src/Symfony/Bridge/Twig/AppVariable.php

```diff
@@ -135,25 +137,25 @@ public function getDebug(): bool
      */
     public function getFlashes(string|array $types = null): array
     {
-        try {
-            if (null === $session = $this->getSession()) {
-                return [];
-            }
-        } catch (\RuntimeException) {
-            return [];
-        }
-
-        if (null === $types || '' === $types || [] === $types) {
-            return $session->getFlashBag()->all();
-        }
+        $flashes = $this->getFlashesUncached($types);
+        $result = $flashes;
 
         if (\is_string($types)) {
-            return $session->getFlashBag()->get($types);
-        }
+            if (isset($this->flashBagCache[$types])) {
+                $result = [...$result, ...$this->flashes[$types]];
```

Comment:
`$this->flashes` does not exist

Reply:
Sorry, I rather copy pasted it in the GitHub file editor. It was a bad idea to refactor stuff in there. New push is done more thoroughly.